server/src/controllers/favorite/addToFavorite.ts

```diff
@@ -3,7 +3,7 @@ import { addToFavoriteService } from '../../services/favorite/favorite.service';
 import { UserNotFoundError } from '../../helpers/errors';
 import { responseHandler } from '../../helpers/responseHandler';
 import { publicProcedure } from '../../trpc';
-import * as validator from '../../middleware/validators/index';
+import * as validator from '../../../../packages/src/validations';
```

Comment:
We should be using the @packrat/packages as a dependency within the server and client, not just doing ../../ to get to the packages directory